e2e-tests/cypress/tests/integration/channels/accessibility/accessibility_sidebar_spec.ts

```diff
@@ -193,50 +194,48 @@ describe('Accessibility tests for RHS getting focus after buttons actions', () =
 
     it('Focus should be on RHS when opening Recent Mentions', () => {
         // # Click the Recent Mentions button
-        cy.findByRole('button', {name: /Recent mentions/i}).click();
+        cy.findByRole('button', {name: /Recent mentions/i}).focus().realPress('Enter');
 
         // * Verify RHS is open
         cy.get('#sidebar-right').should('be.visible');
 
         // * Check that the RHS container is focused
-        cy.get('.sidebar-right-container').should('be.focused');
+        cy.get('.sidebar-right-container').find('button[aria-label="Expand Sidebar Icon"]').should('be.focused');
```

Comment:
Would be better to use `cy.findByLabelText`, ex:
```suggestion
        cy.findByLabelText('Expand Sidebar Icon').should('be.focused');
```

Reply:
done!